### DISCUSSION TITLE
License

### DISCUSSION THREAD

**Discussion by neonima** [devcherchecollegue-org/rust-pong#12] [General]
Quelle license serait appropriée pour le projet ?

Je pensais à MIT license mais si on souhaite commercialisé plus tard il va falloir établir un plan plus claire entre ce qui est public et ce qui ne l'est pas :)

**Comment by Fred-si** [comment]
J’aurais plus pensé à une WTFPL étant donné qu’à la base le projet est juste là pour se faire les jambes avec rust

**Reply by neonima** [reply]
WTFPL ?

**Reply by Fred-si** [reply]
https://en.wikipedia.org/wiki/WTFPL

**Reply by neonima** [reply]
Disons que si on finit par un truc payant, la partie publique serait cool d'avoir une license qui nous mentionne, enfin c'est mon avis :P

**Reply by Westixy** [reply]
https://en.wikipedia.org/wiki/Beerware

**Comment by Fred-si** [comment]
Au final, comme proposé par @Westixy la licence [Beer-ware](https://en.wikipedia.org/wiki/Beerware) me parait plus à propos.

Personnellement si quelqu’un trouve le moyen de gagner de l’argent avec, ça ne me pose pas de problème de ne pas être mentionné, étant donné que j’aurais déjà été payé.

**Comment by neonima** [comment]
Ok on part la dessus!
